osu.Game/Storyboards/Storyboard.cs

```diff
@@ -36,28 +38,22 @@ public StoryboardLayer GetLayer(string name)
         /// <summary>
         /// Whether the beatmap's background should be hidden while this storyboard is being displayed.
         /// </summary>
-        public bool ReplacesBackground(BeatmapInfo beatmapInfo)
+        public bool ReplacesBackground
         {
-            var backgroundPath = beatmapInfo.BeatmapSet?.Metadata?.BackgroundFile?.ToLowerInvariant();
-            if (backgroundPath == null)
-                return false;
+            get
+            {
+                var backgroundPath = BeatmapInfo.BeatmapSet?.Metadata?.BackgroundFile?.ToLowerInvariant();
+                if (backgroundPath == null)
+                    return false;
 
-            return GetLayer("Background").Elements.Any(e => e.Path.ToLowerInvariant() == backgroundPath);
+                return GetLayer("Background").Elements.Any(e => e.Path.ToLowerInvariant() == backgroundPath);
+            }
         }
 
-        public float AspectRatio(BeatmapInfo beatmapInfo)
-            => beatmapInfo.WidescreenStoryboard ? 16 / 9f : 4 / 3f;

```

Comment:
You should make an issue for the re-implementation of this, since this is now broken.

Reply:
It's still implemented to the same level it was (see `CreateDrawable`)

Reply:
Obviously I meant for the background